README.md

```diff
@@ -8,10 +8,30 @@ This package is base on [react-native-ble-manager](https://github.com/innoveit/r
 
 ## Installation
 
+Because this module depends on some node core modules, and react-native doesn't currently have a [resolve.alias a la webpack](https://productpains.com/post/react-native/packager-support-resolvealias-ala-webpack), you will need to use [rn-nodeify](https://github.com/tradle/rn-nodeify) for your shimming needs.
+
+A suggested workflow:
+
+1. Install
+
 ```sh
 yarn add @pagopa/io-react-native-proximity
+
+yarn add --dev patch-package
+yarn add --dev rn-nodeify
```

Comment:
Are these steps required? After adding `patch-package` and `nodeify` it should be enoght a yarn to the root of the repo.

Reply:
This is a suggestion for those who will use the package. Being a specific dependency only for react native it is not within the peer dependencies. So anyone who will use the package if it is on an RN environment will have to follow these steps but they are not valid for others.